zero/gkyl_rect_grid_priv.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
```

Comment:
@jRoeltgen 
i don't think you need a private header just to declare the signature of private functions. Either just place those functions in the .c file before they get used, or put the functions themselves in the private header.

Reply:
although I understand why one may want to do it this way, we just haven't AFAIK, so let's stick to the current model for now.